apps/wrapper/src/main/index.ts

```diff
@@ -33,14 +50,39 @@ function createWindow(): void {
   } else {
     mainWindow.loadFile(join(__dirname, '../renderer/index.html'));
   }
+
+  mainWindow.on('close', async function (e) {
+    if (fileOpened) {
+      childKilled = true;
+      process.kill(pid, 'SIGTERM');
+      setTimeout(async () => {
+        await delFiles();
+      }, 1000);
+    }
+    const choice = dialog.showMessageBoxSync(mainWindow, {
+      type: 'question',
+      buttons: ['Yes', 'No'],
```

Comment:
Make it clearer by using `Confirm` and `Cancel`

Reply:
Fixed